codespell_lib/data/dictionary.txt

```diff
@@ -8928,6 +8928,7 @@ fitering->filtering
 fitler->filter
 fitlers->filters
 fivety->fifty
+fixe->fixed, fixes, fix,
```

Comment:
Suggest fixme.

Reply:
also "fixer"

Reply:
Good points, added both in e5f687a3da1f89e3069adba7dcf0f5d765a930bb